.travis.yml

```diff
@@ -51,3 +51,4 @@ addons:
     branch_pattern: coverity-scan
 
 script: cd src && make && make test
+script: ./kak -e "buffer *debug*; write debug_buffer\n; quit" && echo "*** This is the debug buffer, where debug info will be written ***\n" | cmp debug_buffer -
```

Comment:
I could change it so that it compares it with the content of the debug buffer of kakoune run with -n. But it's not a big deal to change the message string here.
AFAIR, the test suite runs without loading external script (I tried to fix this in the past but it didn't land), I will check.